doc/source/release-notes/Percona-Server-5.6.36-82.1.rst

```diff
@@ -20,43 +20,83 @@ milestone at Launchpad
+
+ |Percona FT| could assert for various reasons deserializing nodes with no useful
+ error output. Bug fixed :tdbbug:`24`.
+
+ |Percona Server| could crash when running a query over a partitioned table that
+ uses an index to read a range of rows if this range was not covered by any
+ existing partition. Bug fixed :bug:`1657941` (upstream :mysqlbug:`76418`).
 
  With two client connections to a server (debug server build), the server could
  crash after one of the clients set the global option ``userstat`` and flushed
- the client statistics (``FLUSH CLIENT_STATICTICS``) and then both clients were
+ the client statistics (``FLUSH CLIENT_STATISTICS``) and then both clients were
  closed. Bug fixed :bug:`1661488`.
 
- When connecting to the Aurora cluster end point using SSL, SAN (Subject
- Alternative Name) certificates were ignored.  Bug fixed :bug:`1673656` (upstream
- :mysqlbug:`68052`).
+ |Percona FT| did not pass cmake flags on to snappy cmake. Bug fixed
+ :tdbbug:`41`.  The progress status for partitioned TokuDB table ALTERs was
+ misleading. Bug fixed :tdbbug:`42`.
 
- Killing a stored procedure which contained a query with a view resulted in a
- debug assertion failure. Bug fixed :bug:`1689736` (upstream :mysqlbug:`86260`).
+ When a client application connecting to the Aurora cluster end point using SSL
+ (``--ssl-verify-server-cert`` option), wildcard and :abbr:`SAN (Subject
+ Alternative Name)` enabled SSL certificates were ignored.  Bug fixed
+ :bug:`1673656` (upstream :mysqlbug:`68052`). See :ref:`compatibility-matrix`.
 
- It was not possible to build |Percona Server| on Debian 9 (stretch) due to
- issues with openssl1.1. Bug fixed :bug:`1702903`.
+ Killing a stored procedure execution could result in an assert failure on a
+ debug server build. Bug fixed :bug:`1689736` (upstream :mysqlbug:`86260`).
 
- The ``SET STATEMENT .. FOR`` statement did not work after ``SET GLOBAL`` or
- ``SHOW GLOBAL STATUS`` commands and affected the global value. Bug fixed
- :bug:`1385352`.
+ It was not possible to build |Percona Server| on Debian 9 (stretch) due to
+ issues with OpenSSL 1.1. Bug fixed :bug:`1702903` (upstream :mysqlbug:`83814`). 
 
- Killing a stored procedure which contained a query with a view resulted in a
- debug assertion failure. Bug fixed :bug:`1689736` (upstream :mysqlbug:`86260`).
+ The ``SET STATEMENT .. FOR`` statement changed the global instead of the session
+ value of a variable if the statement occurred immediately after the ``SET
+ GLOBAL`` or ``SHOW GLOBAL STATUS`` command. Bug fixed :bug:`1385352`.
 
+ The synchronization between the LRU manager and page cleaner threads was not
+ done at shutdown. Bug fixed :bug:`1689552`.
 
 Other bugs fixed: 
+:tdbbug:`6`,
+:tdbbug:`44`,
+:tdbbug:`65`,
 :bug:`1160986`,
 :bug:`1676740`,
 :bug:`1689989`,
 :bug:`1689998`,
 :bug:`1690012`,
 :bug:`1699788`, and
 :bug:`1684601` (upstream :mysqlbug:`86016`).
+
+.. _compatibility-matrix:
+
+Compatibility Matrix
```

Comment:
I have created a ticket to update the documentation: https://jira.percona.com/browse/DOC-572